guard/src/commands/migrate_tests.rs

```diff
@@ -86,4 +89,56 @@ fn test_migrate_rules() -> Result<()> {
     let span = crate::rules::parser::Span::new_extra(&result, "");
     rules_file(span)?;
     Ok(())
+}
```

Comment:
Please add tests for Conditional Rules (CR), rule with `OR`s of CR, rule with `OR`s with Conditional Rule and a Basic rule combination.